src/Containers.jsx

```diff
@@ -33,11 +33,17 @@ class Containers extends React.Component {
     }
 
     deleteContainer(container, event) {
-        event.preventDefault();
```

Comment:
It's good that this gets dropped (it should rarely be used), but just want to ensure that this is deliberate?

Reply:
Yes. I  deleted it deliberately.